src/main/java/com/google/collegeplanner/servlets/PlannerServlet.java

```diff
@@ -39,8 +39,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     try {
       body = getBody(request);
     } catch (ParseException | NullPointerException e) {
-      respondWithError(
-          "Invalid body for POST request.", HttpServletResponse.SC_BAD_REQUEST, response);
+      respondWithError(HttpServletResponse.SC_BAD_REQUEST, response);
       return;
     }
```

Comment:
When you merge master into this branch, there may be one more respondWithError you'll have to change

Reply:
Noted